src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -217,7 +217,7 @@ def remove_vg(vg_name):
     """
     Removes a volume group.
     """
-    fail_msg = "Unable to remove vg %s".format(vg_name)
+    fail_msg = "Unable to remove vg %s" % vg_name
```

Comment:
woah, great catch! thanks for fixing these

Reply:
Heh, thank VS Code